includes/Infrastructure/ServiceBasedPlugin.php

```diff
@@ -396,7 +449,7 @@ function () use ( $class ) {
 	 * @param Injector $injector Injector instance to configure.
 	 * @return Injector Configured injector instance.
 	 */
-	protected function configure_injector( Injector $injector ) {
+	protected function configure_injector( Injector $injector ): Injector {
```

Comment:
This made a problem for me with PHPstan as I recall. Why is this needed?

Reply:
The method takes an Injector and returns an Injector, so it's reasonable to add a return type.

I don't see any PHPStan complaints here and tests pass, so seems all good.